### DISCUSSION TITLE
[feature-request]: snippets support

### DISCUSSION THREAD

**Discussion by hackmajoris** [ghostty-org/ghostty#3871] [Feature Requests, Ideas] [answered]
I used to perform predefined snippets like this:

<img width="416" alt="image" src="https://github.com/user-attachments/assets/ea6e1cef-cf94-4b88-b24a-8c8d983aa9e6" />

Can this be added in the future?

**Comment by pluiedev** [comment, marked as answer]
You can send a predefined sequence of text by doing `keybind = <bind>=text:<text>`
